libexec/rtld-elf/rtld_c18n.c

```diff
@@ -999,26 +978,22 @@ unwind_stack(struct jmp_args ret, void *rcsp, struct trusted_frame *target)
 	return (ret);
 }
 
-struct jmp_args _rtld_longjmp(struct jmp_args, void *, void **);
-struct jmp_args _rtld_unw_setcontext_impl(struct jmp_args, void *, void **);
-
-struct jmp_args
-_rtld_longjmp(struct jmp_args ret, void *rcsp, void **buf)
-{
-	return (unwind_stack(ret, rcsp, cheri_unseal(*buf, sealer_jmpbuf)));
+bool
+c18n_is_enabled(void) {
```

Comment:
```suggestion
c18n_is_enabled(void)
{
```

Inconsistent style with the rest of the functions